pwa/src/components/attributes/attributeTable.tsx

```diff
@@ -5,11 +5,12 @@ import {
   Card,
   Modal
 } from "@conductionnl/nl-design-system/lib";
-import { Link } from "gatsby";
+import {Link} from "gatsby";
```

Comment:
See how nothing actually changed here, but Github is showing the change. This is because you did not yet use `prettier`. Can you please go through all these files and save it again with `prettier` active?